addons/website_slides/static/src/js/slides_embed.js

```diff
@@ -227,22 +227,32 @@ $(function () {
             var widget = $('.oe_slide_js_share_email');
             var input = widget.find('input');
             var slideID = widget.find('button').data('slide-id');
-            if (input.val() && input[0].checkValidity()) {
+            if (input.val()) {
                 widget.removeClass('o_has_error').find('.form-control, .custom-select').removeClass('is-invalid');
                 $.ajax({
                     type: "POST",
                     dataType: 'json',
-                    url: '/slides/slide/send_share_email',
```

Comment:
Since we will re-split the controller methods, will have to see how this can be nicely factorized to avoid too much copy/paste.